specs/pvs/corla.pvs

```diff
@@ -422,6 +455,7 @@ BEGIN
   original_ballot: TYPE FROM ballot
```

Comment:
Line 405/438 ff: it's unclear what we mean, precisely, by ballot. Technically, I doubt we want a single instance of a ballot to refer both to a paper ballot and a digital representation. Needs correction or, if correct, clarification.